source/base/static/base/js/backbone/crawl_models.js

```diff
@@ -92,6 +92,15 @@
       var that = this;
       this.collection = collection;
       this.collectionView = collectionView;
+
+      // If the add crawl modal has been opened at the same time as the add model form,
+      // change focus back to the add crawl modal after the form has been closed
+      // or submitted.
+      $(this.modal).on('hidden.bs.modal', function(event){
+        if ($("#addCrawlModal").attr("aria-hidden") == "false"){
```

Comment:
Basically, what happens is that when you close a modal, it removes the `modal-open` class from the body. If there is another modal open, then it breaks the scrolling for modal that is still open.

If the add crawl modal is open, then we want to re-add that `modal-open` class to make sure that the scrolling doesn't break.

We only want this to happen if the add crawl modal is open, hsnce the check for "aria-hidden"